report/2.Analysis/Requirements.tex

```diff
@@ -0,0 +1,39 @@
+\section{Requirements}
+In order to be able to conclude upon the success of the project, a fixed set of requirements must be formulated.
+These requirements will serve as a guideline in the conclusion of the report and as markers throughout the project. 
+To formally specify the requirements, the MoSCoW analysis model will be utilized. 
+The MoSCoW model specifies which requirements \textbf{M}ust, \textbf{S}hould, \textbf{C}ould, and \textbf{W}on't be fulfilled to deem the project successful.
```

Comment:
jeg ville gerne have tilføjet et par linjer heroppe. Evt kort gennemgang der siger "okay disse problemstillinger har vi, så dem forsøger vi løst. derfor vil vores requirements fokusere på dette. og så trække den lidt sammen med de andre sections måske.
evt sige "vi har baseret på analysen fundet disse ting som relevante". så der er en argumentation bag de forskellige handlinger. 

evt en definition af hvad et target er (Her ville jeg nævne en rød ballon og/eller bold, evt så det er åbent så vi kan skifte hvis det bliver for svært at ramme en bold.)